### DISCUSSION TITLE
Is there a file that contains all the pcb information?

### DISCUSSION THREAD

**Discussion by Nitrous2000** [MarkBryanMilligan/LanternPowerMonitor#44] [General]
I’ve got lots of experience with creating pcb’s, but not familiar with easyEDA. 
IS there a zip file to D/l?
I’ve installed easyEDA, would buy the kits/pcb’s but the store is closed…

thanks folks
Doug

**Comment by Rogero54** [comment]
I have the same question. A bit of a newb at github, but cannot find a place to get the gerber file.

**Comment by DaT1ck** [comment]
EasyEDA will import the JSON file located here:  https://github.com/MarkBryanMilligan/LanternPowerMonitor/tree/main/pcb/EasyEDA

In EasyEDA (online or offline) use File->Open and select the JSON.  Should work fine.
